src/calculateRentalCost.js

```diff
@@ -20,7 +20,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+
+  if (days >= 7) {
+    return cost * days - 50;
```

Comment:
Добавляй скобочки в выражение для улучшения читаемости выражений
```suggestion
    return (cost * days) - 50;
```